recipes/splits/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "splits" %}
+{%set version = "0.1.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "b87dbd579c72e65200a12c7bdf6b01707ec368d2851fa2f5b0ee7a7983bf9fda" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
```

Comment:
It would likely support python 3.x with some trivial patching. I didn't include it because of this error during the import test:
```
===== testing package: splits-0.1.1-py35_0 =====
import: 'splits'
Traceback (most recent call last):
  File "/opt/anaconda3/conda-bld/splits_1474254447005/test_tmp/run_test.py", line 27, in <module>
    import splits
  File "/opt/anaconda3/conda-bld/splits_1474254447005/_t_env/lib/python3.5/site-packages/splits/__init__.py", line 1, in <module>
    import writers
ImportError: No module named 'writers'
TESTS FAILED: splits-0.1.1-py35_0
```